### PR TITLE
Updated documentation for running the contract API tests

### PR DESCRIPTION
**Pull request by chartjes**

### CODE REVIEW

File: docs/dev/api-tests.rst [hunk 1]
```diff
@@ -3,6 +3,24 @@ API Contract Tests
 These tests are designed to look for changes to the recipe server API that are
 not expected.
 
+Before running these tests you will need the following installed:
+
+.. Python 3.6.0 or greater
+.. _virtualenv: https://virtualenv.pypa.io/en/latest/virtualenv installed
```
Comment (mythmon) [inline docs/dev/api-tests.rst]: This isn't the syntax for lists in markdown. I think the right syntax would be

```rst
* Python 3.6.0 or greater
* virtualenv_

.. _virtualenv: https://virtualenv.pypa.io/en/latest/virtualenv
```

That would create a list of two elements, the second of which is a link.

File: docs/dev/api-tests.rst [hunk 2]
```diff
@@ -11,8 +29,8 @@ To run these tests, use the following command from the root project directory.
 
 where ``<server>`` is one of
 
-`https://normandy-admin.stage.mozaws.net`
-`https://normandy-admin.prod.mozaws.net`
+.. `https://normandy.stage.mozaws.net`
+.. `https://self-repair.mozilla.org`
```
Comment (mythmon) [inline docs/dev/api-tests.rst]: List syntax again here.

**Comment by stephendonner** [comment]
@mythmon sure thing; looking now.  And thanks, @chartjes for digging in on this!

**Comment by chartjes** [comment]
On Monday I will update the documentation on TestRail to match these changes

**Comment by mythmon** [comment]
Actually, I realized I can do this myself. So I did.

**Comment by stephendonner** [comment]
Finally slogged through fixing my postgresql installation issue:

```
Warning: postgresql-9.6.2 already installed, it's just not linked.
sdonner-17447:~ sdonner$ brew link postgresql
Linking /usr/local/Cellar/postgresql/9.6.2... 
Error: Could not symlink share/man/man3/SPI_connect.3
/usr/local/share/man/man3 is not writable.
sdonner-17447:~ sdonner$ chown -R sdonner:20 /usr/local/share/man/man3
chown: /usr/local/share/man/man3/Git.3pm: Operation not permitted
chown: /usr/local/share/man/man3/Git::I18N.3pm: Operation not permitted
chown: /usr/local/share/man/man3/Git::SVN::Editor.3pm: Operation not permitted
chown: /usr/local/share/man/man3/Git::SVN::Fetcher.3pm: Operation not permitted
chown: /usr/local/share/man/man3/Git::SVN::Memoize::YAML.3pm: Operation not permitted
chown: /usr/local/share/man/man3/Git::SVN::Prompt.3pm: Operation not permitted
chown: /usr/local/share/man/man3/Git::SVN::Ra.3pm: Operation not permitted
chown: /usr/local/share/man/man3/Git::SVN::Utils.3pm: Operation not permitted
chown: /usr/local/share/man/man3: Operation not permitted
sdonner-17447:~ sdonner$ sudo chown -R sdonner:20 /usr/local/share/man/man3
sdonner-17447:~ sdonner$ brew install postgresql
Warning: postgresql-9.6.2 already installed, it's just not linked.
sdonner-17447:~ sdonner$ brew link postgresql
Linking /usr/local/Cellar/postgresql/9.6.2... 
Error: Could not symlink share/man/man7/ABORT.7
/usr/local/share/man/man7 is not writable.
sdonner-17447:~ sdonner$ sudo chown -R sdonner:20 /usr/local/share/man/man7
sdonner-17447:~ sdonner$ brew link postgresql
Linking /usr/local/Cellar/postgresql/9.6.2... 375 symlinks created
```

**Comment by chartjes** [comment]
@stephendonner I never run the contract API tests against a local copy of Normandy -- I always run them on whatever was deployed to staging. Seemed to me to not be a good use of my time to get it running locally.

**Pull request merged**